index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+
+  <body>
+    <p id="para-1">Hello World we are software developers!</p>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <title id="title">Donald Trump</title>
```

Comment:
The title tag should always be inside the head of the document. It may cause crashes to the site if used within the body. Please position it inside the head tag. 👀